hugegraph-it/hg-it-dist/pom.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
hugegraph-it name is a bit hard to understand, it needs to be more specific, such as hugegraph-cluster-deploy or hugegraph-cluster-test

Reply:
> hugegraph-it name is a bit hard to understand, it needs to be more specific, such as hugegraph-cluster-deploy or hugegraph-cluster-test

+1, already already mentioned it

`hugegraph-mini-cluster` is also fine to me (easy to understand)

Reply:
plan to replace hugegraph-it with hugegraph-cluster-test

Reply:
> the module name have been named for hugegraph-xxx, so the submodule perfect to keep consistent with the name, i mean, the 'hg' should replace with "hugegraph"

maybe refer to the pd/store modules?.... 🔢